kyaml/openapi/kustomizationSwagger.json

```diff
@@ -0,0 +1,442 @@
+{
+  "definitions": {
+    "ConfigMapArgs": {
```

Comment:
The Kubernetes types have a prefix that includes the api group.  We should do something similar.

Reply:
Done.